src-tauri/Cargo.lock

```diff
@@ -1770,12 +1770,6 @@ version = "0.3.16"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "2a60c7ce501c71e03a9c9c0d35b861413ae925bd979cc7a4e30d060069aaac8d"
 
-[[package]]
-name = "minisign-verify"
-version = "0.2.1"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "933dca44d65cdd53b355d0b73d380a2ff5da71f87f036053188bf1eab6a19881"
-
```

Comment:
Same here

Reply:
btw, why is Cargo.lock not in the gitignore?

Reply:
It allows developers to have exact same versions of dependencies

Reply:
It's the same as NPM and `package-lock.json`. Locks dependency to certain version for reproducible builds.